samples/CdnSignedUrlSignatureComputationSample/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace CdnSignedUrlSignatureComputationSample
+{
+    /// <summary>
+    /// Serves as a main entry point into the application.
+    /// </summary>
+    class Program
+    {
+        /// <summary>
+        /// The method invoked as the main entry point for execution.
+        /// </summary>
+        /// <param name="args"> Parameters to compute the signature for signed urls.</param>
+        /// The signature is redirected to the standard output i.e. it is printed on the command line.
+        static void Main(string[] args)
+        {
+            // Validate the arguments passed to the application.
+            if (args.Length < 6)
+            {
+                Console.WriteLine("Error in the application usage.");
+                Console.WriteLine("Usage: ./SignatureComputationApp <ResourcePath> <ExpiresParamName> <ExpiresParamValue> <KeyIdParamName> <KeyIdParamValue> <Secret>");
+                Environment.Exit(1);
+            }
+
+            // Get the signed url signature based on the parameters passed.
+            SignedUrlSignature signedUrlSignature = new SignedUrlSignature();
+
+            string signature = null;
+            try
+            {
+                signature = signedUrlSignature.GetSignature(args[0], args[1], args[2], args[3], args[4], args[5]);
+            }
+            catch (Exception e)
```

Comment:
I'd recommend removing this try/catch and allowing the exception to go unhandled.  The `catch` block is emitting an abnormal termination signal already, and the additional error message doesn't provide additional context to the exception.

Reply:
Fixed!